fastestimator/cli/train.py

```diff
@@ -86,6 +94,7 @@ def configure_train_parser(subparsers: argparse.PARSER) -> None:
                         dest='hyperparameters_json',
                         type=str,
                         help="The path to the hyperparameters JSON file")
+    parser.add_argument('--warmup', dest='warmup', type=str, help="Wheter to perform warmup training, True or False")
```

Comment:
spelling error in help text. Also, this should use action="store_true" instead of being a string that we parse manually

Reply:
done